src/pages/404.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Link from 'gatsby-link';
+import Layout from '../components/layout';
+
+export default function SiteError() {
+  return (
+    <Layout>
+      <h1>404</h1>
+      <p>
+        Oops, look like there&rsquo;s nothing to see here!
+        <br />
+        This page has either been removed or never existed.
+      </p>
+      <p>
```

Comment:
i dont think there should be a need for the link to be wrapped in a paragraph. if simply for spacing, emotion should be available to apply css rules. happy to walk through those details if they would be helpful

Reply:
Emotion isn't in `site` yet. It would need to be normal react styling at this point I think.

Reply:
yeah, i realized the same after posting this. simple styling would probably be fine for the PR. once we merge here, i think we'd probably follow up to extract to `components` fairly shortly afterward, so that would open up more options